### PR TITLE
Updates to CAO and e-sign process

### PR DESCRIPTION
**Pull request by TQuock**

### CODE REVIEW

File: operations/operations/company-agreements.md [hunk 1]
```diff
@@ -6,27 +6,39 @@ Company agreements should only be signed by CEO, Ian Tien, after the approval of
...
+* Aneal Vallurupalli \(Finance\) - sales contracts, corporate contracts, vendor contracts
+* Tim Quock \(Accounting and Procurement\) - sales contracts, corporate contracts, vendor contracts
+* VP \([MLT](https://handbook.mattermost.com/operations/operations/mlt-cadence#mattermost-leadership-team-mlt)\) - vendor contracts
+
+## Who can send contracts for signature?
+
+The following staff are approved to send contracts to be initialed and e-signed
+
+* Lynn Conway  - primary for staff agreements and consulting agreements
+* Amy Nicole - primary for vendor contracts and corproate agreements
```
Comment (it33) [inline operations/operations/company-agreements.md]: 1. Amy Nicol (no "e")
2. "corproate" typo?

File: operations/operations/company-agreements.md [hunk 2]
```diff
@@ -6,27 +6,39 @@ Company agreements should only be signed by CEO, Ian Tien, after the approval of
...
+* Tim Quock \(Accounting and Procurement\) - sales contracts, corporate contracts, vendor contracts
+* VP \([MLT](https://handbook.mattermost.com/operations/operations/mlt-cadence#mattermost-leadership-team-mlt)\) - vendor contracts
+
+## Who can send contracts for signature?
+
+The following staff are approved to send contracts to be initialed and e-signed
+
+* Lynn Conway  - primary for staff agreements and consulting agreements
+* Amy Nicole - primary for vendor contracts and corproate agreements 
+* Kaely Wright - primary for sales agreements and customer supplier forms
```
Comment (it33) [inline operations/operations/company-agreements.md]: Please check people's names in general for this section? "Kaely" doesn't seem right.

File: operations/operations/company-agreements.md [hunk 3]
```diff
@@ -6,27 +6,39 @@ Company agreements should only be signed by CEO, Ian Tien, after the approval of
...
 CAOs by area include: 
 
-* Melissa Warner \(Sales Ops\) 
-* Natalie Jew \(HR\)
-* Aneal Vallurupalli \(Finance\)
-* Tim Quock \(Accounting and Procurement\) 
+* Natalie Jew \(HR\) - employment contracts, benefit contracts, vendor contracts (HR) 
+* Aneal Vallurupalli \(Finance\) - sales contracts, corporate contracts, vendor contracts
+* Tim Quock \(Accounting and Procurement\) - sales contracts, corporate contracts, vendor contracts
+* VP \([MLT](https://handbook.mattermost.com/operations/operations/mlt-cadence#mattermost-leadership-team-mlt)\) - vendor contracts
```
Comment (it33) [inline operations/operations/company-agreements.md]: I don't think MLT should be CAO. 

1. I'd prefer someone from G&A to initial the initial vendor agreement so we have clarity terms for things like indemnity, termination for convenience, jurisdiction, etc. are properly reviewed. 

2. I'm fine having MLT initial in addition to a CAO, for things like spend over $50K, etc. but we need someone trained in contract review as CAO.

**Review by it33 — DISMISSED** [review]
Some changes requested. In addition, could I ask that in this Pull Request we also add something to this page noting that Pull Requests for updates need sign-off by both VP Finance & Ops and CEO in the PR process? 

What's laid out here is critical for keeping the integrity of our review process for company agreements.

File: operations/operations/company-agreements.md (hunk 1 above)
Comment (TQuock) [inline operations/operations/company-agreements.md]: ``
```suggestion
* Amy Nicol - primary for vendor contracts and corporate agreements 
```

File: operations/operations/company-agreements.md (hunk 2 above)
Comment (TQuock) [reply operations/operations/company-agreements.md]:
```suggestion
* Kaelyn Wright - primary for sales agreements and customer supplier forms
```

File: operations/operations/company-agreements.md (hunk 3 above)
Comment (TQuock) [reply operations/operations/company-agreements.md]:
```suggestion

```

File: operations/operations/company-agreements.md (hunk 1 above)
Comment (TQuock) [reply operations/operations/company-agreements.md]:
```suggestion
* Amy Nicol - primary for vendor contracts and corporate agreements 
```

File: operations/operations/company-agreements.md (hunk 2 above)
Comment (TQuock) [reply operations/operations/company-agreements.md]: updated. Was a moving a little too quickly

**Review by TQuock — COMMENTED** [review]
@it33 I have made the pull request https://github.com/TQuock/mattermost-handbook/pull/1#pullrequestreview-478136345

**Comment by it33** [comment]
Looks good to me, thanks Tim!

**Comment by justinegeffen** [comment]
> Some changes requested. In addition, could I ask that in this Pull Request we also add something to this page noting that Pull Requests for updates need sign-off by both VP Finance & Ops and CEO in the PR process?
> 
> What's laid out here is critical for keeping the integrity of our review process for company agreements.

I've updated the page with that note.

**Comment by justinegeffen** [comment]
@TQuock, I've fixed the conflict. Please let me know whether anything else needs to be adjusted. Thanks! :)

**Comment by TQuock** [comment]
@justinegeffen everything looks good from my end

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**